prisma/schema.prisma

```diff
@@ -562,52 +561,65 @@ model EmailAddressEmployee {
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
-  emailAddressId String @db.Uuid
-  emailAddress EmailAddress @relation(fields: [emailAddressId], references: [id])
-  employeeId String @db.Uuid
-  employee Employee @relation(fields: [employeeId], references: [id])
+  emailAddressId String       @db.Uuid
+  emailAddress   EmailAddress @relation(fields: [emailAddressId], references: [id])
+  employeeId     String       @db.Uuid
+  employee       Employee     @relation(fields: [employeeId], references: [id])
 
   @@id([emailAddressId, employeeId])
 }
 
 model BenefitPlan {
-  id String @id @default(uuid()) @db.Uuid
+  id        String   @id @default(uuid()) @db.Uuid
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
   slug String
   name String
 
-  organizationId String @db.Uuid
-  organization Organization @relation(fields: [organizationId], references: [id])
-  employees EmployeeBenefitPlan[]
+  organizationId String                @db.Uuid
+  organization   Organization          @relation(fields: [organizationId], references: [id])
+  employees      EmployeeBenefitPlan[]
 
   @@unique([organizationId, slug])
 }
 
 model CustomField {
-  id String @id @default(uuid()) @db.Uuid
+  id        String   @id @default(uuid()) @db.Uuid
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
-  name String
-  type String
-  required Boolean
-  dateFormat String?
-  decimals Int?
-  enumOptions Json? @db.JsonB
+  name        String
+  type        String
+  required    Boolean
+  dateFormat  String?
+  decimals    Int?
+  enumOptions Json?   @db.JsonB
 
   userId String @db.Uuid
-  user User @relation(fields: [userId], references: [id])
+  user   User   @relation(fields: [userId], references: [id])
 }
 
 model Options {
-  id String @id @default(uuid()) @db.Uuid
+  id        String   @id @default(uuid()) @db.Uuid
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
   options Json? @db.JsonB
 
   userId String @db.Uuid
-  user User @relation(fields: [userId], references: [id])
+  user   User   @relation(fields: [userId], references: [id])
+}
+
+model Department {
+  id        String   @id @default(uuid()) @db.Uuid
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  departmentCode String @unique
+  departmentName String
+
+  // Does each account need their own version of this list?
```

Comment:
@colinfrederickson @bigcountrycrane does each org need a copy of their own departments? Or is this fine to share app-wide?

Reply:
fine to share app-wide at the moment!